tests/spdx/test_cli.py

```diff
@@ -0,0 +1,45 @@
+import os
+
+import pytest
+from click.testing import CliRunner
+
+from spdx.clitools.pyspdxtools import main
+
+
+@pytest.mark.parametrize(
+    "options",
+    [
+        ("--infile", os.path.join(os.path.dirname(__file__), "data/SPDXJSONExample-v2.3.spdx.json")),
+        ("-i", os.path.join(os.path.dirname(__file__), "data/SPDXJSONExample-v2.3.spdx.json"), "--novalidation"),
+        (
+            "-i",
+            os.path.join(os.path.dirname(__file__), "data/SPDXJSONExample-v2.3.spdx.json"),
+            "--novalidation",
+            "--version",
+            "SPDX-2.3",
+        ),
+        ("-i", os.path.join(os.path.dirname(__file__), "data/SPDXJSONExample-v2.3.spdx.json"), "-o", "-"),
+    ],
+)
+def test_cli(options):
+    runner = CliRunner()
+
+    result = runner.invoke(main, options)
+
+    assert result.exit_code == 0
+
+
+@pytest.mark.parametrize(
+    "options",
+    [
+        (),
+        ("-i", os.path.join(os.path.dirname(__file__), "data/SPDXJSONExample-v2.3.spdx.json"), "--version"),
+        ("-i", os.path.join(os.path.dirname(__file__), "data/SPDXJSONExample-v2.3.spdx.json"), "-o"),
+    ],
+)
+def test_cli_with_system_exit(options):
```

Comment:
Sorry, one other thing; every CLI call has a system exit, the important part here is the exit code:
```suggestion
def test_cli_with_system_exit_code_2(options):
```

Reply:
maybe also adapt the other test name to be consistent